### PR TITLE
cloc: v1.66 upgrade

### PR DESCRIPTION
**Pull request by syscrusher**
Simple update for cloc to v1.66

### All Submissions:

- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### CODE REVIEW

**Comment by ilovezfs** [comment]
why is the Makefile not being used?

**Comment by syscrusher** [comment]
Fixed.
I simply didn't notice the new Makefile and it installs fine without it, albeit without a man page.

File: Library/Formula/cloc.rb [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+  url "https://github.com/AlDanial/cloc/archive/v1.66.tar.gz"
+  sha256 "6d8b5e9fac8934478a0fd815f60d814b93d4a12344c4820f5b47a55048b21d53"
   head "https://github.com/AlDanial/cloc.git"
 
   bottle :unneeded
 
   def install
-    bin.install "cloc"
+    cd "Unix" do
+        system "make", "prefix=#{prefix}", "install"
```
Comment (mikemcquaid) [inline Library/Formula/cloc.rb]: Two-character indent. Also, rather than doing `cd` you can pass `"-C", "Unix"`

File: Library/Formula/cloc.rb [hunk 2]
```diff
@@ -1,14 +1,16 @@
 class Cloc < Formula
   desc "Statistics utility to count lines of code"
   homepage "https://github.com/AlDanial/cloc/"
-  url "https://github.com/AlDanial/cloc/archive/1.64.tar.gz"
-  sha256 "4ebac0ee3124df0c5394410f6803ed2b4f82de9035f62a0b89d8ce54e0726709"
+  url "https://github.com/AlDanial/cloc/archive/v1.66.tar.gz"
+  sha256 "6d8b5e9fac8934478a0fd815f60d814b93d4a12344c4820f5b47a55048b21d53"
   head "https://github.com/AlDanial/cloc.git"
 
   bottle :unneeded
```
Comment (mikemcquaid) [inline Library/Formula/cloc.rb]: Remove this line once we use `make`, thanks!

**Comment by syscrusher** [comment]
Changes made. Thanks for the feedback @mikemcquaid.

**Comment by nijikon** [comment]
Merged. Thank you for your contribution to Homebrew!

**Pull request closed without merging**